langchain/src/load/index.ts

```diff
@@ -157,13 +157,19 @@ async function reviver(
     // Construct the object
     if (serialized.type === "constructor") {
       // eslint-disable-next-line new-cap, @typescript-eslint/no-explicit-any
-      return new (builder as any)(
+      const instance: Serializable = new (builder as any)(
         mapKeys(
           kwargs as SerializedFields,
           keyFromJson,
           combineAliasesAndInvert(builder)
         )
       );
+
+      // Minification in severless/edge runtimes will mange the
+      // name of classes presented in traces. As the names in import map
+      // are present as-is even with minification, use these names instead
+      instance.lc_name_override = name;
```

Comment:
Worth just setting the regular `builder.name` property with Object.defineProperty ?